scripts/monitoring/cron-send-node-labels-status.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+""" Node label check for OpenShift V3 """
+
+# Adding the ignore because it does not like the naming of the script
+# to be different than the class name
+# pylint: disable=invalid-name
+
+# pylint: disable=wrong-import-position
+# pylint: disable=broad-except
+# pylint: disable=line-too-long
+
+import argparse
+import time
+
+import logging
+logging.basicConfig(
+    format='%(asctime)s - %(relativeCreated)6d - %(levelname)-8s - %(message)s',
+)
+logger = logging.getLogger()
+logger.setLevel(logging.WARN)
+
+# Our jenkins server does not include these rpms.
+# In the future we might move this to a container where these
+# libs might exist
+#pylint: disable=import-error
+
+from openshift_tools.monitoring.ocutil import OCUtil
+from openshift_tools.monitoring.metric_sender import MetricSender
+
+ocutil = OCUtil()
+
+def runOCcmd(cmd, base_cmd='oc'):
+    """ log commands through ocutil """
+    logger.info(base_cmd + " " + cmd)
+    oc_time = time.time()
+    oc_result = ocutil.run_user_cmd(cmd, base_cmd=base_cmd, )
+    logger.info("oc command took %s seconds", str(time.time() - oc_time))
+    return oc_result
+
+def runOCcmd_yaml(cmd, base_cmd='oc'):
+    """ log commands through ocutil """
+    logger.info(base_cmd + " " + cmd)
+    ocy_time = time.time()
+    ocy_result = ocutil.run_user_cmd_yaml(cmd, base_cmd=base_cmd, )
+    logger.info("oc command took %s seconds", str(time.time() - ocy_time))
+    return ocy_result
+def get_type(hostname):
+    """get the host type , is the host is a master or node or a infra"""
+    host_type = 'compute'
+    if 'master' in hostname:
+        host_type = 'master'
+    elif 'infa' in hostname:
+        host_type = 'infra'
+    else:
+        host_type = 'compute'
+
+    return host_type
+
+def check_label_on_host(labels):
+    """according to the host type to check if the host missed any labels"""
+    result = True
+    hostname = labels['hostname']
+    host_type = labels['type']
+    # if the node miss the hostname and type label
+    if hostname and host_type:
+        pass
+    else:
+        return False
+
+    master_labels = {'beta.kubernetes.io/arch':'amd64', 'beta.kubernetes.io/instance-type': 'm4.xlarge', 'beta.kubernetes.io/os': 'linux', 'failure-domain.beta.kubernetes.io/region': 'us-east-1', 'failure-domain.beta.kubernetes.io/zone': 'us-east-1a', 'hostname': 'tke-master-9f206', 'kubernetes.io/hostname': 'ip-10-147-203-48.ec2.internal', 'network.openshift.io/not-enforcing-egress-network-policy':'true', 'node-role.kubernetes.io/master': 'true', 'region': 'us-east-1', 'type': 'master'}
```

Comment:
Do we need check all the labels? I don't think every node will have label 'region': 'us-east-1' .